migration-engine/migration-engine-tests/tests/existing_data/mod.rs

```diff
@@ -140,7 +140,7 @@ async fn altering_a_column_with_non_null_values_should_warn(api: &TestApi) -> Te
 
     api.schema_push(dm2).send().await?.assert_warnings(&[
         match api.sql_family() {
-            SqlFamily::Postgres => "The `age` column on the `Test` table would be dropped and recreated. This will lead to data loss.".into(),
+            SqlFamily::Postgres => "The `age` column on the `Test` table would be dropped and recreated. This will lead to data loss.",
```

Comment:
weird, I can't imagine how this is affected by additional targets (these are just regular cargo tests that run on all targets)

Reply:
yeah, not sure what caused that heh.